tools/api-markdown-documenter/src/test/snapshots/markdown/simple-suite-test/default-config/test-suite-a/testinterfacewithcallsignature-interface.md

```diff
@@ -0,0 +1,35 @@
+# TestInterfaceWithCallSignature
+
+[Packages](/) > [test-suite-a](/test-suite-a/) > [TestInterfaceWithCallSignature](/test-suite-a/testinterfacewithcallsignature-interface)
+
+An interface with a complex call signature.
+
+<h2 id="testinterfacewithcallsignature-signature">Signature</h2>
+
+```typescript
+export interface TestInterfaceWithCallSignature
+```
+
+## Call Signatures
+
+| Call Signature | Return Type | Description |
+| - | - | - |
+| [\<T extends string>(foo: T): number & T](/test-suite-a/testinterfacewithcallsignature-interface#_call_-callsignature) | number & T | Test call signature. |
+
+## Call Signature Details
+
+<h3 id="_call_-callsignature"><T extends string>(foo: T): number & T</h3>
```

Comment:
Note for reviewers: this illustrates a bug that I just discovered, which was not previously covered by the test suite. I will have a fix up for it following this PR.